packages/databyss-services/pages/index.js

```diff
@@ -0,0 +1,28 @@
+import { httpGet, httpPost } from '../lib/requestApi'
+
+// TODO: Add native versions of these
+export function setAccountId(value) {
```

Comment:
The accountId get/set functions should live in `databyss-services/auth/index.js` alongside the the token get/set functions